gift-list-export.user.js

```diff
@@ -13,6 +13,12 @@
 (function () {
   'use strict'
 
+  // Update these based on your registry before running the script
+  const REGISTRY_ID = '2MSXVAPN0SUAA';
+  const PRIMARY_OWNER_CUSTOMER_ID = 'A45579H563N1Z';
+  const REGISTRY_SHIPPING_ADDRESS_ID = 'A3R5EH9ZY1D1OZ';
+  const OUTPUT_DELIMITER = ',';
+
```

Comment:
Awesome. I wonder if we can eventually make a single request/response to grab most of these values